app/plugins/Files/js/helpers.js

```diff
@@ -3,7 +3,7 @@
 // Library for communicating with Sia-UI
 const IPC = require('ipc');
 // Library for arbitrary precision in numbers
-const BigNumber = require('bignumber.js');
+const BigNumber = require('../../js/bignumber.min.js');
```

Comment:
Didn- DIDN'T HAVE NODEJS? It's a Nodejs project -____-
Hope it's not too limiting/messy in the future...